common/lib/dependabot/clients/azure.rb

```diff
@@ -153,8 +153,9 @@ def create_commit(branch_name, base_commit, commit_message, files,
           "/pushes?api-version=5.0", content.to_json)
       end
 
+      # rubocop:disable Metrics/ParameterLists
       def create_pull_request(pr_name, source_branch, target_branch,
-                              pr_description, labels)
+                              pr_description, labels, work_item)
```

Comment:
Somewhat annoying but just thinking about existing users who might be calling these methods directly so should probably make these optional so we don't introduce breaking changes

```suggestion
                              pr_description, labels, work_item = nil)
```